frontend/src/pages/projects/screens/detail/overview/components/OverviewCard.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import {
+  Card,
+  CardHeader,
+  Flex,
+  FlexItem,
+  Popover,
+  Text,
+  TextContent,
+} from '@patternfly/react-core';
+import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
+import { ProjectObjectType, SectionType } from '~/pages/projects/types';
+import HeaderIcon from '~/pages/projects/components/HeaderIcon';
+import DashboardPopupIconButton from '~/concepts/dashboard/DashboardPopupIconButton';
+
+import './OverviewCard.scss';
+
+type OverviewCardProps = {
+  objectType: ProjectObjectType;
+  sectionType: SectionType;
+  title?: string;
+  popoverHeaderContent?: string;
+  popoverBodyContent?: string;
+  children: React.ReactNode;
+  id?: string;
+};
+const OverviewCard: React.FC<OverviewCardProps> = ({
+  objectType,
+  sectionType,
+  title,
+  popoverHeaderContent,
+  popoverBodyContent,
+  children,
+  id,
+}) => (
+  <Card id={id} className={css('odh-overview-card', sectionType)} style={{ height: '100%' }}>
```

Comment:
Why are we assigning an ID here and to the title text?

Reply:
An ID needs to be passed to the title text so that we can reference it in the `aria-labelledby` attribute of the "view all" buttons in the Workbenches and Pipelines cards on the "overview" page. It allows the accessible name to be more unique for both, so instead of two "view all" buttons, there'll be "view all workbenches" and "view all pipelines" buttons